packages/mouse-trap/src/MouseTrap.js

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import debounce from 'lodash.throttle';
```

Comment:
**nitpick**: I think that it would be good to fix this name to match the import so that we can remember which one we want in the future.
```suggestion
import throttle from 'lodash.throttle';
```